Classes/Authentication/Common/AccountType/ViewModels/AccountTypeViewModel.swift

```diff
@@ -24,11 +24,19 @@ struct AccountTypeViewModel: PairedViewModel {
     private(set) var title: EditText?
     private(set) var detail: EditText?
     private(set) var badge: String?
+    private(set) var shouldShowNewAccountWarning: Bool = false
     
     init(_ model: AccountSetupMode) {
         bindImage(model)
         bindTitle(model)
         bindDetail(model)
+        
+        switch model {
+        case .addBip39Address:
+            shouldShowNewAccountWarning = true
+        default:
```

Comment:
Try to avoid using `default` case. In the case where `AccountSetupMode` will be extended with additional case the compiler will be unable to compile the code before we check and adopt this `switch`. This will reduce the number of cases where we forgot about some `switch` in the future.